src/ergw_aaa_session.erl

```diff
@@ -316,14 +309,10 @@ handle_event(info, {'$reply', Promise, Handler, Msg, Opts} = _Info,
     aaa_state_stats(Handler, State, StateOut),
     Data = Data0#data{handlers = maps:put(Handler, StateOut, HandlersS), session = SessOut},
     update_session(SessOut, EvsOut, Data),
-    {keep_state, Data};
-
-handle_event(Type, Event, _State, _Data) ->
-    ?LOG(warning, "unhandled event ~p:~p", [Type, Event]),
-    keep_state_and_data.
+    {keep_state, Data}.
 
 terminate(Reason, Data) ->
-    ?LOG(error, "ctld Session terminating with state ~p with reason ~p", [Data, Reason]),
+    ?LOG(debug, "terminating with state ~p with reason ~p", [Data, Reason]),
```

Comment:
Would not a log level of **info** be better here?
Perhaps I am not up to date with the meaning of log levels.

Reply:
a call to `terminate` is a completely normal thing. I don't see how this would justify anything else then a `debug`.